modules/database/database.py

```
@@ -0,0 +1,151 @@
+        :param webhook_type: The type of webhook to get
+        :type webhook_type: TautulliWebhookTrigger
+        :param minutes: The number of minutes to look back
+        :type minutes: int
+        :return: A list of webhooks of the specified type received in the past x minutes
+        """
+        timestamp = get_minutes_ago_timestamp(minutes=minutes)
+        try:
+            return self._get_all_entries(Webhook,
+                                         (Webhook.webhook_type == webhook_type),
+                                         (Webhook.created_at >= timestamp))  # type: ignore
+        except Exception as e:
```

Ruff (0.8.0)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)